minerl/env/core.py

```diff
@@ -668,8 +668,19 @@ def _init_mission(self):
                 num_retries += 1
                 if num_retries > MAX_WAIT:
```

Comment:
MAX_WAIT is probably longer than the time it takes for Minecraft to start up. We don't really need 80 retries, so we could probably adjust this downwards.

Reply:
I'd like to be somewhat conservative with the number of retries because I'm hoping to launch possibly a few dozen instances at once, in which case it might take longer for any one Minecraft instance for load.

When I forcibly launched exactly two instances on the same port without detecting BindExceptions, IIRC it took less than 2 minutes to finish all the retries.